t/t3428-rebase-edit-todo.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/sh
+
+test_description='rebase should reread the todo file if an exec modifies it'
+
+. ./test-lib.sh
+
+todo=$(git rev-parse --git-dir)/rebase-merge/git-rebase-todo
+
+test_expect_success setup '
+
+	>file &&
+	git add file &&
+	test_tick &&
+	git commit -m "Initial commit"
+'
+
+test_expect_success 'rebase exec modifies rebase-todo' "
+        git rebase HEAD -x 'echo x touch F >>\"$todo\"' &&
```

Comment:
It took me a moment to realize that the `x` is not a place holder here. It would most likely make sense to stave off such unnecessary confusion by spelling out `exec` explicitly.

Also: The quotes around `$todo` are unnecessary, too (otherwise the assignment above would have needed quotes), therefore it would make a lot of sense to lose them here, to make the thing more readable.